update the CHANGELOG (#95)
knurling-rs/probe-run: CHANGELOG.md

## [Unreleased]

### Added

- [#30] added a `--no-flash` flag to run a program without re-flashing it

> [#30]: https://github.com/knurling-rs/probe-run/pull/30

Author: (I wish we had those asciidoctor macros used in the Rust-Analyzer blog post)

- [#40] added (`--help`) documentation to the many CLI flags

> [#40]: https://github.com/knurling-rs/probe-run/pull/40

Reviewer: Interleaving them like this results in more spacing between the entries. Could you either update the new entries to not do that, or make the old entries have the same spacing?

Author: I wrote them like this because it make it easier to not miss adding the `[#12]: https` line. Unfortunately, it seems the link doesn't render properly if there's no newline between the bullet and the outline link. I can cluster the links again.

- [#55] added more precise stack overflow detection for apps linked with `flip-link`

[#55]: https://github.com/knurling-rs/probe-run/pull/55

- [#57] added module location information to log messages

[#57]: https://github.com/knurling-rs/probe-run/pull/57

- [#63] added file location information to the stack backtrace

[#63]: https://github.com/knurling-rs/probe-run/pull/63

- [#83] added git info to the `--version` output

[#83]: https://github.com/knurling-rs/probe-run/pull/83

- [#88] added `--speed` flag to set the frequency of the probe

[#88]: https://github.com/knurling-rs/probe-run/pull/88

### Fixed

- [#28] notify the user ASAP that RTT logs were not found in the image

[#28]: https://github.com/knurling-rs/probe-run/pull/28

- [#50] fixed a bug that was causing an infinite stack backtrace to be printed

[#50]: https://github.com/knurling-rs/probe-run/pull/50

- [#51] fixed the handling of Ctrl-C

[#51]: https://github.com/knurling-rs/probe-run/pull/51

- [#77] flush stdout after each write; fixes a bug where output was not printed until a newline was sent from the device using non-defmt-ed RTT

[#77]: https://github.com/knurling-rs/probe-run/pull/77

### Changed

> - [#25] increased RTT attach retries. Needed, in some cases, for inter-operation with `rtt-target`

Reviewer: ```suggestion
- [#25] increased RTT attach retries, which is sometimes needed for inter-operation with `rtt-target`
```
